vars/githubWorkflowRun.txt

```diff
@@ -0,0 +1,113 @@
+Run workflow on github actions
+
+### Run as step:
+
+```
+  Map runInfo = githubWorkflowRun(repo: "owner/repository", workflow: "build.yml", ref: "main",
+    parameters: [path: "filebeat"], credentialsId: "github-workflow-token")
+```
+
+#### Run asyncronous:
+
+```
+  script {
+    Map args = [
+       repo: "owner/repository",
+       workflow: "build.yml", 
+       ref: "main",
+       parameters: [path: "filebeat"],
+       credentialsId: "github-workflow-token"]
+    int runId = githubWorkflowRun.triggerGithubActionsWorkflow(args)
+    Map runInfo = githubWorkflowRun.getWorkflowRun(args + [runId: runId]) 
+  }
+
+```
+
+### Arguments:
+
+* repo: repositiry owner and name
+* workflow: workflow file name
+* ref: reference ( branch, tag or hash)
+* parameters: map with parameters to pass to the workflow 
+
+### Returs:
+
+runInfo : information about run
+
+
+### Workflow script example:
```

Comment:
IMO, I'm not sure if this section is needed, but to mention the section regarding what's needed to be configured in the GitHub workflow to be able to use the `githubWorkflowRun` step correctly.